### PR TITLE
[Spanish] localize 'Policy as Code (PaC)'

### LINKED ISSUE #1401: [Spanish] localize 'Policy as Code (PaC)'
Localize the `content/en/policy-as-code.md` document to Spanish

### PR DESCRIPTION
**Pull request by tinniaru3005**
### Describe your changes
Localize the `content/en/policy-as-code.md` document to Spanish

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1401 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/.wordlist.txt [hunk 1]
```diff
@@ -142,6 +142,7 @@ on
 on-premises
 Open
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:143]:
```suggestion
Open
PaC
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
It seems like your local `dev-es` branch is not properly updated, and it keeps bringing some changes from `main`

File: content/es/policy-as-code.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Política como código (PaC)
+status: Comentarios apreciados
+category: concepto
```
Comment (electrocucaracha) [inline content/es/policy-as-code.md:4]:
```suggestion
category: Concepto
```

**Comment by tinniaru3005** [comment]
> It seems like your local `dev-es` branch is not properly updated, and it keeps bringing some changes from `main`

When I am forking the repository I am only able to fork from the `main` branch. Then I am creating my own branch and working on it. Can you please tell me a way I can form from the `dev-es` branch

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha, can you please tell me if the changes are fine now?

**Review by electrocucaracha — APPROVED** [review]

**Review by ramrodo — APPROVED** [review]
/lgtm

**Review by raelga — APPROVED** [review]

**Comment by krol3** [comment]
Hi @tinniaru3005 it's only missing the DCO to merge it

**Pull request merged**